RefrigeratorExe/RefrigeratorExe/RefrigeratorExe.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
very good work with git. it's good that you included all relevant files so I could run the exe. very well.